src/seismometer/configuration/model.py

```diff
@@ -353,3 +420,10 @@ def _reduce_derived_features(derived_features: list[Event | Cohort], field_name=
             seen_names.add(feature.display_name)
             good_features.append(feature)
         return good_features
+
+
+def _search_dictionary(dictionary: list[DictionaryItem], key: str) -> Optional[DictionaryItem]:
+    for item in dictionary:
+        if item.name == key:
+            return item
+    return
```

Comment:
Can we raise a KeyError since a key is missing?

Reply:
Done